### PR TITLE
<CustomModal> - testkit type fixes

### PR DESCRIPTION
**Pull request by itaytay**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: .wuf/testkits/enzyme-typescript.template.ejs [hunk 1]
```diff
@@ -5,6 +5,7 @@ import AvatarDriver from '../src/Avatar/Avatar.uni.driver';
 import {BadgeDriver} from '../src/Badge/Badge.driver';
 import {BadgeSelectDriver} from '../src/BadgeSelect/BadgeSelect.driver';
 import {BoxDriver} from '../src/Box/Box.uni.driver';
+import {CustomModalDriver} from '../src/CustomModal/CustomModal.uni.driver';
```
Comment (simast) [inline .wuf/testkits/enzyme-typescript.template.ejs]: note that you also need to add this to a map below:

https://github.com/wix/wix-style-react/blob/e0661bb24b942ab44e03f776093d1dada925cec9/.wuf/testkits/enzyme-typescript.template.ejs#L33-L45

File: .wuf/testkits/vanilla-typescript.template.ejs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import {BaseUniDriver} from 'wix-ui-test-utils/base-driver';
 import {BaseDriver} from 'wix-ui-test-utils/driver-factory';
+import {CustomModalDriver} from '../src/CustomModal/CustomModal.uni.driver';
```
Comment (simast) [inline .wuf/testkits/vanilla-typescript.template.ejs]: same (please add to the map below)

Comment (itaytay) [reply .wuf/testkits/vanilla-typescript.template.ejs]: Done

File: .wuf/testkits/enzyme-typescript.template.ejs (hunk 1 above)
Comment (itaytay) [reply .wuf/testkits/enzyme-typescript.template.ejs]: Done

**Pull request merged**